qa/L0_parameters/test.sh

```diff
@@ -97,10 +97,64 @@ for i in "${all_tests[@]}"; do
   wait $SERVER_PID
 done
 
+
+# Test Classification Extension
+MODELDIR="models"
+TEST_RESULT_FILE="test_results.txt"
+TEST_SCRIPT_PY="./class_count_test.py"
+
+rm -rf $MODELDIR
+mkdir -p "${MODELDIR}/identity_fp32/1"
+cp /opt/tritonserver/qa/python_models/identity_fp32/config.pbtxt "${MODELDIR}/identity_fp32/"
+cp /opt/tritonserver/qa/python_models/identity_fp32/model.py "${MODELDIR}/identity_fp32/1/"
```

Comment:
Interesting approach. 
I see changed relative path in favor to hardcoded value, it creates literally the same behavior. 
My assumption was that you will create a `TEST_WORKDIR` variable or similar, it will have the default location and can be overwritten if needed.

Reply:
Thank you, got it. I made a change [8744c17](https://github.com/triton-inference-server/server/pull/8267/commits/8744c177b2aaaf59e13d3acf6550856edf64d86a) for the same.